src/main/java/com/prgrms/kream/domain/member/controller/MemberController.java

```diff
@@ -1,8 +1,32 @@
 package com.prgrms.kream.domain.member.controller;
 
+import static org.springframework.http.HttpStatus.*;
+
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.prgrms.kream.common.api.ApiResponse;
+import com.prgrms.kream.domain.member.dto.MemberRegisterRequest;
+import com.prgrms.kream.domain.member.dto.MemberRegisterResponse;
+import com.prgrms.kream.domain.member.facade.MemberFacade;
+
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequestMapping("/api/v1/member")
+@RequiredArgsConstructor
 public class MemberController {
 
-	public int getI() {
-		return 1;
+	private final MemberFacade memberFacade;
+
+	@PostMapping("")
```

Comment:
```suggestion
	@PostMapping
```
이렇게 바꾸면 더 보기 좋을 것 같아요 :)

Reply:
수정했습니다!